tests/integration/test_charm.py

```diff
@@ -78,6 +78,88 @@ async def test_setup_discourse(
     assert response.status_code == 200
 
 
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+async def test_db_migration(model: Model, ops_test: OpsTest, pytestconfig: Config, run_action):
+    """
+    arrange: preload postgres with a mock db
+    act: deploy and integrate with discourse
+    assert: discourse must be active idle
```

Comment:
I feel like this isn't quite describing what we're doing fully. We load a v3.2.0 DB which has contents that would trigger a bug (can we mention what bug as a reference here), and then we deploy discourse v3.3.0 and verify that it handles the schema migration correctly.

Reply:
I can write it in a more detailed way, thank you!